web/stories/django-girls-bucaramanga.rst

```diff
@@ -247,25 +247,70 @@ contigo para coordinar los detalles.
    <div class="section-guias">
 
 .. template:: bootstrap3/django-girls-guia
-   :name: Diego Fernando Acosta Ortiz
-   :image: guia-diego.png
+   :name: Andrea Pérez Becerra
+   :image: guia-dennise.png
    :place: Bucaramanga, Colombia
-   :community: Python Bucaramanga
-   :web: http://cusol.uis.edu.co/
-   :github: https://github.com/dacostaortiz/
-   :facebook: https://www.facebook.com/dfacos
+   :email: andrea.perez.be@gmail.com
```

Comment:
I don't like this. I mean, we shouldn't publish other people's email. I think that social networks is different because is something public, but not sure about the email.